spring-web/src/test/java/org/springframework/web/client/MessageBodyClientHttpResponseWrapperTests.java

```diff
@@ -0,0 +1,98 @@
+package org.springframework.web.client;
```

Comment:
Please add the missing copyright header with the date range set to 2002-2021.

See `HttpMessageConverterExtractorTests` for an example.